site/content/entry/yuzu-progress-report-may-2021/index.md

```diff
@@ -0,0 +1,355 @@
+when building yuzu with any of the valid compilers, be it MSVC, GCC, Clang or even MinGW.
+## UI and other Quality of Life improvements
+
+Users have the option to customize the game list in `Emulation > Configure… > General > UI`, the Add-Ons column can be disabled there. 
+Previously, if the user disabled it, the game list was not being refreshed, forcing the user to find inventive ways to manually do it.
+Thanks to [Kewlan](https://github.com/Kewlan)’s work, 
+[now the game list properly refreshes itself with no need for manual intervention!](https://github.com/yuzu-emu/yuzu/pull/6298)
+
+{{< imgs
+	"./add-ons.png| Add-Ons shows which update, DLCs and mods are installed and enabled"
+  >}}
+
+toastUnlimited reluctantly [added the CPU tab to per-game settings.](https://github.com/yuzu-emu/yuzu/pull/6321) At the same time, Custom RTC was removed to force it as a global value.
+
+{{< imgs
+	"./cpu.png| Totally not a hint of anything"
+  >}}
+
+Now, while we’re on this subject, some things that need to be clarified about the CPU settings tab. 
+Unsafe was originally only intended for CPUs that lacked the FMA instruction set, which causes games to run at very low framerates.
+Later on, a fix was discovered that could boost the performance of `Luigi’s Mansion 3` by reducing precision. This was described in 
+[January’s progress report.](https://yuzu-emu.org/entry/yuzu-progress-report-jan-2021/)
+
+As a result, we recommend users of CPUs that do have FMA to stick to `Accurate` and only force `Unsafe` for Luigi’s Mansion 3. 
+Using Unsafe is known to cause precision issues, for example, exaggerating the hitboxes of characters in `Super Smash Bros. Ultimate`.
+Users with old or low-end Intel CPUs that lack FMA, feel free to keep Unsafe enabled all the time.
+
+We provide some examples in the following beautiful and totally perfect table:
+
+| CPU series | FMA support | Recommended setting |
+| :-----------: | :--------------: | ------------ |
+| Intel Core 4000 to 11000 series | Yes | Use Accurate, Unsafe for LM3 |
+| Intel Core 500 to 3000 series and older | No | Use Unsafe |
+| Intel Atom, Celeron and Pentium series | No | Use Unsafe |
+| AMD Ryzen series | Yes | Use Accurate, Unsafe for LM3 |
+| AMD FX and A 4000 to 9000 APU series | Yes | Use Accurate, Unsafe for LM3 |
+| AMD Phenom, A 3000 APU series and older | No | Use Unsafe |
+
+*LM3 = Luigi’s Mansion 3
+
+toastUnlimited also gave us the option to [apply settings while games are running](https://github.com/yuzu-emu/yuzu/pull/6346). 
+This can help more comfortably test for ideal settings while playing.
+Keep in mind that settings that require a reboot to take effect, won't be affected by the apply button.
+
+{{< imgs
+	"./apply.png| "
+  >}}
+
+Continuing with the per-game setting improvements, toast also added an exception for homebrews.
+Since almost all homebrews use a game-ID of 0000000000000000, using the ID to save the per-game settings would have been useless, so instead, 
```

Comment:
```suggestion
Since most homebrew use a game-ID of 0000000000000000, using the ID to save the per-game settings would have been useless, so instead, 
```